include/mxnet/gpu_swap_history.h

```diff
@@ -0,0 +1,56 @@
+#ifndef GPU_SWAP_HISTORY_H_
+#define GPU_SWAP_HISTORY_H_
+
+#include <chrono>
+#include <vector>
+#include "storage.h"
```

Comment:
Change to "./storage.h" to be consistent with MXNet's coding style. Also, please put this line after #include<mutex>

Reply:
Btw, is this include necessary?

Reply:
I first put the definition of record_t and timestamp_t in storage.h, but not that they are in swap.h I can directly include swap.h instead. By "put this line after #include" do you mean that comment?

Reply:
After all #include <xxx>. I like seeing them in some order, first #include <xxx> then #include " xxx".